index.js

```diff
@@ -28,7 +28,10 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  //could also make a new array, loop through movies and push movie.title into new array
+  return movies.map((movie) => movie.title);
```

Comment:
Nice job!  Technically on a one liner like this we don't even need the `()` around movie.